src/.gitignore

```diff
@@ -0,0 +1 @@
+WCSimRootDict.cxx
```

Comment:
Not sure how you've got a `.cxx`!? The GNUmakefile uses `.cc` Maybe a Root6 thing?
But should be consistent - use the `$WCSIMDIR/.gitignore`, or move these two lines (without `src/`) from that file into this one
```
src/WCSimRootDict.cc
src/WCSimRootDict.h
```